FIRO_TSEnsembles/src/main/java/hec/ensemble/stats/Total.java

```diff
@@ -1,11 +1,12 @@
 package hec.ensemble.stats;
 
-import static hec.ensemble.stats.ConvertUnits.convertCfsAcreFeet;
+import javax.measure.IncommensurableException;
+import javax.measure.Unit;
 
 public class Total implements Computable, Configurable {
     private static final String DEFAULT_INPUT_UNITS = "cfs";
     Configuration config;
-    private String outputUnit;
+    private Unit<?> unit = null;
```

Comment:
Is the null initializer of this global variable necessary?

Reply:
Removed initializer